ci/delete-packages-from-space/build.gradle.kts

```diff
@@ -0,0 +1,209 @@
+import io.ktor.client.*
+import io.ktor.client.engine.jetty.*
+import kotlinx.coroutines.runBlocking
+import org.apache.commons.io.FilenameUtils
+import space.jetbrains.api.runtime.*
+import space.jetbrains.api.runtime.resources.projects
+import space.jetbrains.api.runtime.types.*
+import java.util.*
+import kotlin.collections.ArrayList
+
+val checkJavaVersion = tasks.register("checkJavaVersion") {
+    doLast {
+        check(JavaVersion.current() >= JavaVersion.VERSION_1_9) {
+            "Use JDK 9+ to run this task"
+        }
+    }
+}
+
+tasks.register("listProjectsAndPackageRepositories") {
+    dependsOn(checkJavaVersion)
+
+    doLast {
+        Space().listProjectsAndPackageRepositories()
+    }
+}
+
+val packagesToDeleteFile: File
+    get() = project.buildDir.resolve("packages-to-delete.txt")
+
+tasks.register("generateListOfPackagesToDelete") {
+    dependsOn(checkJavaVersion)
+    doLast {
+        Space().preparePackagesToDelete(packagesToDeleteFile)
+    }
+}
+
+tasks.register("deletePackages") {
```

Comment:
Please add some interactive prompt telling what and from where is deleted, and allowing to stop deletion in the last moment.

Reply:
OK, if file has to be edited - fine with me.